### PR TITLE
Adding basic mailer class for notifications

### PR DESCRIPTION
**Pull request by arfon**
Related: https://github.com/openjournals/theoj/issues/148

### CODE REVIEW

**Comment by arfon** [comment]
@marcrohloff @freelanceastro for now I've set up a single email forward `robot@theoj.org` which forwards to my this [test inbox](https://mailinator.com/inbox.jsp?to=theoj). Sendgrid is set up on Heroku so we should be good to go with these configs.

File: app/mailers/notification.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Notification < ActionMailer::Base
+  default :from => "robot@theoj.org"
```
Comment (marcrohloff) [inline app/mailers/notification.rb]: Just as a general rule I'm trying to use the new ruby hash style i.e. ```from:'robot@theoj.org'```

Comment (arfon) [inline app/mailers/notification.rb]: Fair enough. I try to keep to the old one :smile:

I'll update the PR to be consistent with your style.

**Comment by arfon** [comment]
@marcrohloff - better?

**Pull request merged**